config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/DiggingDeeper-AAAAAAAAAAAAAAAAAAAF2Q==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Now that you know where to dig, dig all the way down to bedrock to find if there is a vein located there.\n\nNow, what if you know what you want but not where to find it? Our fancy bartimaeusnek developer has added a nifty feature to NEI that shows you where to find ores! Simply look for the oreblock in NEI, and click on it for info on where to find that ore.\n\nThere are many types of ores, even just for getting one material like Iron. So don\u0027t stick to only one type.\n\nYou can also search small ores, but not all of them are used."
+      "desc:8": "Now that you know what an Ore Chunk is, dig all the way down to bedrock to find if there is a vein located in it.\n\nCheck out an ore in NEI to see where you can find a vein that contains it. Search for it and click on it to get this info, among other things.\n\nThere are many types of ores, even just for getting one material like Iron, so don\u0027t stick to only one ore.\n\nYou can also search small ores, but they are spawned at random and not all of them are used."
```

Comment:
"they are spawned at random and"

this is just wrong.

Reply:
Thank you! I replaced it with "scattered".

Reply:
i mean I guess they are that but isnt this line about searching in NEI?

Reply:
Ok, got it. I addressed it in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/0def8d43e3072957dfcd87e142909df2f06e9913